recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,297 @@
+import os
+import shutil
+import stat
+from glob import glob
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PythonOption:
+    OFF = "off"
+    SYSTEM = "system"
+    # in future we may allow the user to specify a version when
+    # libPython is available in Conan Center Index.
+    ALL = [OFF, SYSTEM]
+
+
```

Comment:
For `remove_files_by_mask`

```suggestion


required_conan_version = ">=1.29.1"

```